test/runtests.sh

```diff
@@ -29,12 +28,17 @@ Options
  -x FX_EXECUTABLE    path to Firefox executable (default: $FX_EXECUTABLE)
  -d                  enable debug logging
  -c                  open JavaScript console and don't quit on completion
+ -t                  run only translator-dependent tests
  TESTS               set of tests to run (default: all)
 DONE
 	exit 1
 }
 
-while getopts "x:dc" opt; do
+DEBUG=false
+UPDATE_TRANSLATORS=true
+TRANSLATOR_TESTS=false
```

Comment:
Only disabled for zotero/zotero in Travis. Explained in https://github.com/zotero/zotero/issues/699#issuecomment-94552471